draft-ietf-quic-transport.md

```diff
@@ -3471,6 +3471,9 @@ Datagrams containing Initial packets MAY exceed 1200 bytes if the client
 believes that the Path Maximum Transmission Unit (PMTU) supports the size that
 it chooses.
 
+UDP datagrams MUST NOT be fragmented at the IP layer.  In IPv4
```

Comment:
That is not a requirement we can enforce, and it does not seem to be within QUIC's purview to require it. How about the following: "QUIC implementations are expected to ensure that UDP Datagrams containing QUIC packets are not fragmented by the IP layer at the sender. In IPv4 {{...}}, the DF bit MUST be set to prevent fragmentation on the path."